pkg/config/sink.go

```diff
@@ -168,15 +168,18 @@ func (s *SinkConfig) applyParameter(sinkURI *url.URL) error {
 	}
 
 	// validate that protocol is compatible with the scheme
-	if isMqScheme(sinkURI.Scheme) {
+	if IsMqScheme(sinkURI.Scheme) {
 		var protocol Protocol
 		err := protocol.FromString(s.Protocol)
 		if err != nil {
 			return err
 		}
+	} else if s.Protocol == "default" {
```

Comment:
Why we need this?

Reply:
To ease restrictions appropriately.

Reply:
But shouldn't it be fixed right at startup?